packages/account/package.json

```diff
@@ -28,7 +28,7 @@
     },
     "dependencies": {
         "@binary-com/binary-document-uploader": "^2.4.7",
-        "@deriv/api-types": "^1.0.54",
+        "@deriv/api-types": "^1.0.11",
```

Comment:
why do we need to change `"@deriv/api-types"` version?

Reply:
Because it is failing some of the test cases in codecov and it doesn't have that specific call unless you update it.